### ISSUE TITLE
Bug on latest firefox (56.0)

### ISSUE THREAD

**Issue by gitrequests** [gibbok/animatelo#5]
Problem with **flipInX** and **flipInY** animations (maybe more, i tested their only).

Not show animation (animated block hides from page), when no mouse move and no keyboard press button (New firefox throttling?).
When mouse moving it works correct.

[jsfiddle](https://jsfiddle.net/xkritikx/jxz6r03p/)

Animations without opacity works correct.
Tested on the 2 PC (Windows 10, Firefox 56 not working, Chrome 61 works, Opera 48 works).

Update:
Fade (**fade**, **fadeInDown** and etc.) effects with opacity works great. At the moment the problem  only with flipInX and flipInY animations.

**Comment by gibbok** [comment]
Thanks for reporting, very interesting bug.
I double check the code of animatelo.js and at a first look I do not see any issues there.
This problem exists when using `setInterval` and `requestAnimationFrame` when trying to create an animation loop wrapping animatelo (Which under the hood use Wen Animation API).
IMO it is specific of the newest release of FF 56 and possibly related to Web Animation API and the event loop (we should report it to Mozilla team).

By the way... the correct way to control an animation in animatelo.js is to use its [JS api](https://github.com/gibbok/animatelo/blob/master/usage.md) (which also works on the latest FF).

Live:
https://jsfiddle.net/cd86mb4v/

API Example:
```
  window.animatelo.flipInX('#animate', {
    iterations: Infinity,
    duration: 2000,
    direction: 'normal',
    fill: 'forwards',
  });
```
You can use `iterations: Infinity` to get rid off `setInterval` and execute correctly your animation.
In this way you bypass the FF issue using properly the API.
I hope it helped.
Thanks.

**Comment by gitrequests** [comment]
Firefox fix it on 57.0 release (works for me)!
